ocs_ci/ocs/ui/base_ui.py

```diff
@@ -195,6 +202,53 @@ def take_screenshot(self):
         self.driver.save_screenshot(filename)
         time.sleep(0.5)
 
+    def do_clear(self, locator, timeout=30):
+        """
+        Clear the existing text from UI
+
+        Args:
+            locator (tuple): (GUI element needs to operate on (str), type (By))
+            timeout (int): Looks for a web element until timeout (sec) occurs
+
+        """
+        wait = WebDriverWait(self.driver, timeout)
+        element = wait.until(ec.element_to_be_clickable((locator[1], locator[0])))
+        element.clear()
+
+    def wait_until_expected_text_is_found(self, locator, expected_text, timeout=300):
```

Comment:
Please change the timeout to 10 which will be used for most scenarios and customize it in your call

Reply:
I have kept it to 60 by default